lib/openstudio-standards/standards/necb/NECB2011/service_water_heating.rb

```diff
@@ -1,10 +1,10 @@
 class NECB2011
-  def model_add_swh(model:, swh_fueltype: 'DefaultFuel', shw_scale:)
+  def model_add_swh(model:, shw_fueltype: 'DefaultFuel', shw_scale:)
```

Comment:
keep swh, not shw - that aligns with the method model_add_swh and the module. change throughout

Reply:
Revised. Now we only have swh_fueltype as suggested.